modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java

```diff
@@ -785,6 +786,30 @@ public void setHasDiscriminatorWithNonEmptyMapping(boolean hasDiscriminatorWithN
         this.hasDiscriminatorWithNonEmptyMapping = hasDiscriminatorWithNonEmptyMapping;
     }
 
+    @Override
+    public boolean getIsString() { return isString; }
```

Comment:
why are getters one line, but setters 3 lines?

Reply:
Just copying and pasting. Do you have a preference on how it should be done?

Reply:
>Do you have a preference on how it should be done?

Consistent :)